src/providers/irc/IrcServer.hpp

```diff
@@ -21,6 +21,10 @@ class IrcServer : public AbstractIrcServer
     const QString &userFriendlyIdentifier();
 
     bool hasEcho() const;
+    /**
+     * @brief sends a private/direct message to target
+     */
+    void sendDirectly(QString target, QString message);
```

Comment:
This should be renamed to something more clear like `sendDirectMessage`,  `sendPrivateMessage`, `sendWhisper` would be the names I'm OK with.